guides/11_Priority_Based_Downlink_Policy.md

```diff
@@ -59,6 +61,18 @@ Be aware of the following when calling this function:
 **Receiving notifications that a remote video was paused due to bandwidth constraint**
 If a remote video source is paused due to insufficient bandwidth, then the application will be notified through the existing notification mechanism by setting the VideoTileState to pausedDueToBandwidth. Note that there is already a member named `paused` in VideoTileState and that is used to track if the application paused the remote video.
 
+**Configure network event delays**
+Remote video source will be paused or unpaused based on bandwidth fluctuation. To configure the delay to pause/unpause, pass in two float values between _0 to 1_ (representing network issue response factor and recovery factor respectively) in initialization of `VideoPriorityBasedPolicyConfig`, and then assign the config to `VideoPriorityBasedPolicy`. The closer to 0 the value is, the smaller the delay is, and vice versa. Use the following setter to change the config dynamically:
+
+```ts
+setVideoPriorityBasedPolicyConfigs(config: VideoPriorityBasedPolicyConfig): void
+```
+
+You can also use the following presets according to usage:
+* *Default Preset*: When no specific use cases is defined. You want to select default preset for most of time. It is used when a `VideoPriorityBasedPolicyConfig` is not given during the initialization of `VideoPriorityBasedPolicy`.
+* *Unstable Network Preset*: If expecting a lot of mobile usage, builders can choose unstable network preset. It has less confidence in the network so it takes longer wait when network is recovered in case the recovery is just a spike.
```

Comment:
Its not clear what knobs we are turning here and that these two presets are changing different ones.  Should clarify.